### PR TITLE
Add files with pip requirements for images odoo90 and odoo100

### LINKED ISSUE #226: Add requirements file to dependencies of repositories
Add a file to focus the python dependencies of repositories for versions 9.0 and 10.0

### PR DESCRIPTION
**Pull request by mapuerta**
This PR resolves fix #226

### CODE REVIEW

File: odoo90/scripts/build-image.sh [hunk 1]
```diff
@@ -32,26 +32,7 @@ NPM_DEPENDS="less \
...
-                   flake8 \
-                   pylint-mccabe \
-                   PyWebDAV \
-                   mygengo \
-                   recaptcha-client \
-                   egenix-mx-base \
-                   hg+https://bitbucket.org/birkenfeld/sphinx-contrib@default#egg=sphinxcontrib-youtube&subdirectory=youtube \
-                   git+https://github.com/vauxoo/pylint-odoo@master#egg=pylint-odoo \
-                   git+https://github.com/vauxoo/panama-dv@master#egg=ruc"
+PIP_DEPENDS_EXTRA=$(cat ${DEPENDENCIES_FILE})
```
Comment (LuisAlejandro) [inline odoo90/scripts/build-image.sh]: Be careful with this.

If you have a `requirements.txt` with comments like these (which are normal):

```
req1
# this req is for Y
req2
```

Then the result of cat'ing that file in a subshell will be:

```
req1 # this req is for Y req2
```

Which is not what you wanted.

**Comment by mapuerta** [comment]
@ruiztulio Ready for review

This is the PR has the change we agreed to in meeting. To add a file with the python dependencies to improve the times in the deployv build

Create the following [script](https://gist.github.com/mapuerta/839068d3023e9fb402bb02620cd62448) to test whether pip dependencies were installed

You run it like that
`python test_issue_227.py --build --rm
`

**Comment by ruiztulio** [comment]
@mapuerta can you add the [dos2unix](https://git.vauxoo.com/vauxoo/instance/blob/10.0/.gitlab-ci.yml#L47) dependencie to the odoo-shippable image too please?

@moylop260 these dependencies are actual dependencies from the some customer's instances (extracted from 10.0 and 9.0 production instances), we are adding them here to speed the build process a little more. Do you think it can be a good idea to do the same with the 8.0 one?

**Comment by ruiztulio** [comment]
@moylop260 ok, as the 8.0 require a little more work since we have to update the odoo shippable dependencies too I'd suggest to work it in different PR (also we need a little more time to colect all 8.0 pip dependencies a generate the proper file)

@mapuerta did you updated the 9.0 files with the latest 8.0 changes? (they are basically a clone, only difference is the dependencies). The update process is just copy & paste the files and change 8 to 9, then apply the dependencies changes. Also, can you check that the abastotal image can be build and the app installed using the resulting 9.0 image?

**Comment by mapuerta** [comment]
@ruiztulio Ready for review

1. And add the changes from 8.0 to 9.0
2. With this [script](https://gist.github.com/mapuerta/839068d3023e9fb402bb02620cd62448) try the installed dependencies in the images 9.0 and 10.0
3. Create an abastotal build with the image generated with the changes

build_test
log_build
These were the times
times

**Comment by ruiztulio** [comment]
@mapuerta did you installed the abastotal app using that image also?

**Comment by mapuerta** [comment]
@ruiztulio YES. run this [script](https://gist.github.com/mapuerta/fbeef98dc4df611831e3805db500358a) to build and install the abastotal app and installed without any problem

log_intalled

These were the time
poc_test_build

**Pull request merged**